integration-tests/gradle/src/integrationTest/kotlin/org/jetbrains/dokka/it/gradle/TestedVersions.kt

```diff
@@ -2,30 +2,36 @@ package org.jetbrains.dokka.it.gradle
 
 internal object TestedVersions {
 
-    val LATEST = BuildVersions("7.4.2", "1.7.20")
-    val BASE =
+    val LATEST = BuildVersions("7.4.2", "1.8.0")
+
+    /**
+     * All supported Gradle/Kotlin versions, including [LATEST]
```

Comment:
Can you put an URL to the compatibility matrix?

Reply:
Good call, I've added two links:

* Link to the [Kotlin/Gradle compatibility matrix](https://docs.gradle.org/current/userguide/compatibility.html#kotlin) 
* Link to [AGP/Gradle compatibility matrix](https://developer.android.com/studio/releases/gradle-plugin#updating-gradle)